assets/model_monitoring/components/src/data_quality_statistics/compute_data_quality_statistics.py

```diff
@@ -3,18 +3,20 @@
 
 """This file contains the core logic for data drift compute metrics component."""
 
-import pyspark
-import pyspark.pandas as ps
 from pyspark.context import SparkContext
 from pyspark.sql.functions import col
 from pyspark.sql.session import SparkSession
 from pyspark.sql.types import StructType, StructField, StringType
+import pyspark
+import pyspark.pandas as ps
 
 
 # Init spark session
 sc = SparkContext.getOrCreate()
 spark = SparkSession(sc)
 
+supported_datatype_for_max_min_value = ["IntegerType()", "DoubleType()", "ByteType()", "LongType()", "FloatType()"]
```

Comment:
do we have any warnings for the non-support max-min which should be used for out bound metrics, right?

Reply:
No. It does not add much value for warning in our driver logs because we will have a list of metrics generated as output. The user can view the metrics for each feature in the output easily, they will notice that some feature does not have outofbands easily.